apps/site/pages/fr/about/governance.md

```diff
@@ -24,7 +24,7 @@ Un guide pour les collaborateurs est disponible à l'adresse suivante : [collabo
 
 ## Comité de pilotage technique
 
-Le projet est régi par le [Comité de pilotage technique (TSC)][technical steering committee (tsc)]
+Le projet est régi par le \[Comité de pilotage technique (TSC)]\[]
```

Comment:
this links keeps getting translated erroneously - i wonder if it has to do with the parser getting confused by the extra `(` `)` in the content

Reply:
it's French I'm checking

Reply:
my point being that it shouldnt be translated, because the link is broken

![image](https://github.com/user-attachments/assets/2dc05873-8d5a-440e-a547-501c8212234b)

Reply:
you should re-sync. I had updated the string to not translate the link

Reply:
I want #7114 to land first - then we can do that